read_json.cpp

```diff
@@ -282,23 +282,38 @@ std::vector<mvt_layer> parse_layers(FILE *fp, int z, unsigned x, unsigned y, int
 			dv = fix_polygon(dv, false, false);
 		}
 
+		// handle longitude wraparound
+		//
+		// this is supposed to be data for a single tile,
+		// so any jump from the left hand side edge of the world
+		// to the right edge, or vice versa, is unexpected,
+		// so move it to the other side.
+
+		if (fix_longitudes) {
+			for (size_t i = 0; i < dv.size(); i++) {
+				if (i > 0) {
+					if ((dv[0].x < (1LL << 30)) &&
```

Comment:
Let's add comments here explaining how the math ties into "quarters of world" logic.